extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -751,9 +848,11 @@ export class AmpPanZoom extends AMP.BaseElement {
       if (this.scale_ <= 1) {
```

Comment:
shouldn't this be `if (this.scale_ <= this.minScale_)`?

Reply:
Yeah you're right. I was under the impression that we fixed this in a previous PR but maybe somewhere a bad merge happened. 0.0